### ISSUE TITLE
[💡 FEATURE REQUEST]: Chinese translation

### ISSUE THREAD

**Issue by shellphy** [roadrunner-server/roadrunner#1964]
### Plugin

None

### I have an idea!

I want to make a Chinese version of the document in order to better promote the project, How to start

**Comment by rustatian** [comment]
Hey @shellphy 👋 
This would be a nice feature to add. However, without native speakers of the language, that would be impossible to implement.
If you can help with that, basically, all the docs are here in MD's: https://github.com/roadrunner-server/docs. I guess that they can be translated via things like ChatGPT and then corrected by the native speaker. 

After that, you may attach the archive here, to the ticket and ping me or join our Discord server and send directly to me.

**Comment by shellphy** [comment]
> Hey @shellphy 👋 This would be a nice feature to add. However, without native speakers of the language, that would be impossible to implement. If you can help with that, basically, all the docs are here in MD's: https://github.com/roadrunner-server/docs. I guess that they can be translated via things like ChatGPT and then corrected by the native speaker.
> 
> After that, you may attach the archive here, to the ticket and ping me or join our Discord server and send directly to me.

As a Chinese, I think such a good project as roadrunner is not very popular in China, which is largely because there is no Chinese version of the document.

**Comment by rustatian** [comment]
Yeah, agree. I would greatly appreciate it if you could help to translate into Chinese.
